test/governance/gov.test.ts

```diff
@@ -0,0 +1,131 @@
+import { expect } from 'chai'
+import hre from 'hardhat'
+import '@nomiclabs/hardhat-ethers'
+
+import { GraphGovernance } from '../../build/typechain/contracts/GraphGovernance'
+
+import { deployProxyAdmin, deployGraphGovernance } from '../lib/deployment'
+import { getAccounts, Account, randomHexBytes } from '../lib/testHelpers'
+
+const { ethers } = hre
+const { HashZero } = ethers.constants
+
+enum ProposalResolution {
+  Null,
+  Accepted,
+  Rejected,
+}
+
+describe.only('GraphGovernance', () => {
```

Comment:
We're using eslint [`no-only-tests`](https://www.npmjs.com/package/eslint-plugin-no-only-tests) to prevent against accidentally committing these - it happened a lot!

Reply:
Good advise :)